t/18-qemu-options.t

```diff
@@ -156,6 +157,71 @@ EOV
 
 };
 
+# Test QEMUTPM option with:
+# * 'instance'
+# * a number
+subtest qemu_tpm_option => sub {
+
+    open(my $var, '>', 'vars.json');
+    print $var <<EOV;
+{
+$common_options
+   "QEMUTPM" : "instance"
+}
+EOV
+    close($var);
+    # call isotovideo with QEMUTPM=instance
+    system("perl $toplevel_dir/isotovideo -d qemu_disable_snapshots=1 2>&1 | tee autoinst-log.txt");
```

Comment:
You know, you can just supply the additional qemutpm parameter as command line, if you prefer